src/test/resources/com/puppycrawl/tools/checkstyle/checks/regexp/regexpsingleline/InputRegexpSinglelineSemantic10.java

```diff
@@ -0,0 +1,19 @@
+/*
+RegexpSingleline
+format = SYSTEM\\.(OUT)|(ERR)\\.PRINT(LN)?\\(
+message =
+ignoreCase = true
+minimum = (default)0
+maximum = (default)0
+fileExtensions = (default)all files
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.regexp.regexpsingleline;
+
+public class InputRegexpSinglelineSemantic10 {
+    public static void main(String[] args) {
+        System.out.println("str"); // violation
```

Comment:
Can we use `// violation ''`
?

To make it clear that message via empty.

Reply:
> Can we use // violation ''
 ?

Yes, I updated the code